db/schema.rb

```diff
@@ -121,7 +141,7 @@
   add_index "instance_users", ["instance_id", "user_id"], name: "index_instance_users_on_instance_id_and_user_id", unique: true
 
   create_table "read_marks", force: :cascade do |t|
-    t.integer  "readable_id",   foreign_key: false
```

Comment:
This is an issue of schema_plus https://github.com/SchemaPlus/schema_plus/issues/155
we need to manually add this back after each migration.